app/src/main/java/eu/kanade/tachiyomi/widget/ExtendedNavigationView.kt

```diff
@@ -269,7 +272,18 @@ open class ExtendedNavigationView @JvmOverloads constructor(
                 is MultiStateHolder -> {
                     val item = items[position] as Item.MultiStateGroup
                     val drawable = item.getStateDrawable(context)
-                    holder.text.setText(item.resTitle)
+
+                    // SY -->
+                    try {
+                        holder.text.setText(item.resTitle)
+                    } catch (e: Exception) {
+                        val sourceManager: SourceManager = Injekt.get()
+                        sourceManager.getCatalogueSources().filter { it.id.toInt() == item.resTitle }.map {
+                            holder.text.text = it.name
+                        }
+                    }
```

Comment:
I wouldnt allow this, maybe make a different Holder for sources that returns the source name